arm-storage/2015-06-15/swagger/storage.json

```diff
@@ -343,14 +324,11 @@
         ],
         "responses": {
           "200": {
-            "description": "",
+            "description": "OK -- list of keys retrieved and returned successfully.",
```

Comment:
sorry I am repeating the same comment. I want to keep it consistent with the latest, either we need to change both or we leave this as is